docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,96 @@ keywords:
   - temporal cloud nexus
 ---
 
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
 :::tip SUPPORT, STABILITY, and DEPENDENCY INFO
 
-Temporal Nexus is available in [Public Preview](https://docs.temporal.io/evaluate/development-production-features/release-stages#public-preview) for both self-hosted deployments and Temporal Cloud with the [Temporal Go SDK](https://docs.temporal.io/develop/go).
+**Temporal Nexus** is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for both [Temporal Cloud](/cloud/nexus) and [self-hosted deployments](/production-deployment/self-hosted-guide/nexus).
+- [Temporal Go SDK](/develop/go/nexus) support for Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview).
+- [Temporal Java SDK](/develop/java/nexus) support for Nexus is available in [Pre-release](/evaluate/development-production-features/release-stages#pre-release).
 
 :::
 
-Temporal Nexus is a feature of the Temporal platform designed to connect durable executions across team, Namespace, region, and cloud boundaries.*
-It promotes a more modular architecture for sharing a subset of your team’s capabilities through well-defined service API contracts for other teams to use, that abstract underlying Temporal primitives, like Workflows, or execute arbitrary code.
+Nexus allows you to reliably connect your Temporal Applications.
+It was designed with Durable Execution in mind and enables each team to have their own Namespace for improved modularity, security, troubleshooting, and fault isolation.
+
+Nexus Services can be used within and across Namespaces, which are connected by a global mTLS-secured Envoy mesh in Temporal Cloud.
+Nexus Operations are arbitrary-duration operations (sync or async) that can be implemented with Temporal primitives, like Workflows, or execute arbitrary code.
```

Comment:
I agree that this sentence could be reworked. The intro is super strong and a great change. I feel like it's digestible and perfect for the Evaluate section. I'm wondering what's another way to say this?

Reply:
> Maybe they can span any length of execution 

Thanks, will go with this.

> Is this the right place to be talking about execution details (Workflows or arbitrary code, which is confusing here)?

This is a key thing that Nexus does, making it easy to abstract and share Workflows, and other Temporal primitives, as Nexus Operations. Backing a Nexus Operation with a Workflow is what makes the Operation durable, since Nexus Operations are not intrinsically durable, they just enable connecting durable executions (Workflows and other primitives).

It's also important to highlight that Nexus Operations are not just for abstracting and sharing Temporal primitives, but can be used to execute arbitrary code like a database query or micro-service call, and can essentially be a Temporal-native micro-service that uses the Temporal queue-based Worker architecture.

Given that, I'm currently leaning towards leave the last part about how Nexus Operations can be implemented, and indicate it's not just a rigid "map your workflows" framework, but gives you the flexibility to use arbitrary code.